src/components/ArtifactSummaryCard.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <p-card class="artifact-summary-card" :class="classes.root">
+    <div class="artifact-summary-card__body" :class="classes.body">
+      <header class="artifact-summary-card__header" :class="classes.header">
+        <h6 class="artifact-summary-card__subheader" :class="classes.subheader">
+          {{ artifactLabel }}
+        </h6>
+        <h3 v-if="artifact.key">
+          {{ artifact.key }}
+        </h3>
+
+        <template v-if="hasRun">
+          <p-bread-crumbs :crumbs="crumbs" class="artifact-summary-card__bread-crumbs" :class="classes.breadCrumbs" />
+        </template>
+      </header>
+
+      <div class="artifact-summary-card__summary-container" :class="classes.summaryContainer">
+        <div
+          class="artifact-summary-card__summary-item"
+          :title="artifact.updated.toLocaleString()"
+        >
+          <span class="artifact-summary-card__summary-item-label">
+            {{ localization.info.artifactUpdatedTimestampLabel }}
+          </span>
+          <span class="artifact-summary-card__summary-item-value">
+            {{ formatDateTime(artifact.updated) }}
+          </span>
+        </div>
+
+        <div
+          class="artifact-summary-card__summary-item"
+          :title="artifact.created.toLocaleString()"
+        >
+          <span class="artifact-summary-card__summary-item-label">
+            {{ localization.info.artifactCreatedTimestampLabel }}
+          </span>
+          <span class="artifact-summary-card__summary-item-value">
+            {{ formatDateTime(artifact.created) }}
+          </span>
+        </div>
+      </div>
+    </div>
+
+    <div v-if="slots.default" class="artifact-summary-card__slot">
+      <slot />
+    </div>
+  </p-card>
+</template>
+
+<script lang="ts" setup>
+  import { BreadCrumbs } from '@prefecthq/prefect-design'
+  import { computed, ref, useSlots } from 'vue'
+  import { useFlowRun, useTaskRun, useWorkspaceRoutes } from '@/compositions'
+  import { localization } from '@/localization'
+  import { Artifact } from '@/models'
+  import { formatDateTime } from '@/utilities'
+
+  const props = defineProps<{
+    artifact: Artifact,
+    condense?: boolean,
+  }>()
+
+  const slots = useSlots()
+  const routes = useWorkspaceRoutes()
+
+  const flowRunId = ref(props.artifact.flowRunId)
+  const taskRunId = ref(props.artifact.taskRunId)
+
+  const flowRun = useFlowRun(flowRunId)
+  const taskRun = useTaskRun(taskRunId)
+
+  const hasRun = computed(() => props.artifact.flowRunId || props.artifact.taskRunId)
+
+  const artifactLabel = computed(() => {
+    return props.artifact.key ? localization.info.artifactLabel : localization.info.artifactResultLabel
+  })
+
+  const crumbs = computed<BreadCrumbs>(() => {
+    const internalCrumbs: BreadCrumbs = []
+
+    if (flowRunId.value) {
+      internalCrumbs.push({
+        text: flowRun.value?.name ?? '|',
```

Comment:
Why the pipe default? Won't there always be a name on a flow run?

Reply:
Meant that to be an empty string - there will always be a name on the flow run (if one exists) but the flow run is async so there's no guarantee it's here